# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextProviderImpl.h
# Commit: 204b899f436d
# Full Hash: 204b899f436d99f0eae759af3ccfcbaabfbbb428
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 3f5baab34b1f -r 204b899f436d gfx/gl/GLContextProviderImpl.h
--- a/gfx/gl/GLContextProviderImpl.h	Thu Jun 11 06:36:55 2020 +0000
+++ b/gfx/gl/GLContextProviderImpl.h	Thu Jun 11 06:37:35 2020 +0000
@@ -66,12 +66,12 @@
    * @return Context to use for offscreen rendering
    */
   static already_AddRefed<GLContext> CreateOffscreen(
-      const mozilla::gfx::IntSize& size, const SurfaceCaps& minCaps,
-      CreateContextFlags flags, nsACString* const out_failureId);
+      const mozilla::gfx::IntSize& size, const GLContextCreateDesc&,
+      nsACString* const out_failureId);
 
   // Just create a context. We'll add offscreen stuff ourselves.
   static already_AddRefed<GLContext> CreateHeadless(
-      CreateContextFlags flags, nsACString* const out_failureId);
+      const GLContextCreateDesc&, nsACString* const out_failureId);
 
   /**
    * Create wrapping Gecko GLContext for external gl context.